.dockerignore

```diff
@@ -14,5 +14,4 @@ nosetests.xml
 coverage.xml
 *,cover
 *.log
-.git
```

Comment:
:o what did this fix? do we need it in worker too?

Reply:
nope, if we're ignoring .git we don't have any information about this directory being a git repository so we wouldn't be able to run any git commands, `git submodule` fails because we aren't in a git repo in the container when `.git` is ignored